lib/views/feedbacks_slack_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Aprb.Views.FeedbacksSlackView do
+
+  def render(event) do
+    %{
+      text: "#{event["subject"]["display"]} #{event["verb"]} #{event["properties"]["message"]} from #{event["properties"]["url"]}",
```

Comment:
Did you try this new view? We read `attachments` and `unfurl_links` from this render response [here](https://github.com/artsy/aprb/blob/master/lib/services/event_service.ex#L15) when calling slack, both of them will be `nil` in this view, i don't know how Slack api deals with `nil` for both of these. If they don't handle it well, we can set defaults for those two fields.